Add MCP Integration docs (#8028)
DevExpress/devextreme-documentation: concepts/Common/Integration Guides/05 DevExpress MCP Server Configuration/20 Prerequisites and Installation.md

The DevExpress MCP server supports the following IDEs and AI tools:

**IDEs**:

- Visual Studio 2022 (version 17.14 or later)
- Visual Studio Code
- Cursor IDE
- JetBrains Rider 2025.1+

**AI tools**:

- GitHub Copilot
- JetBrains AI Assistant
- Claude/Claude Code
- Other MCP-compatible AI Assistants

> [note] We strongly recommend you keep your development tools up to date to ensure access to the latest MCP features.

Reviewer: ```suggestion
[note] We recommend that you keep your development tools up to date so you can access the latest MCP features.
```
